src/EMS/RingdownStatus.js

```diff
@@ -124,12 +135,23 @@ function RingdownStatus({ className, onStatusChange, ringdown }) {
               type="warning"
               title="Cancel Delivery?"
               message="Patient data will be erased and hospital will be notified."
-              cancel="Maintain delivery"
-              destructive="Confirm cancel delivery"
+              cancel="Keep status"
+              destructive="Update status"
               onDestructive={handleDiversionUpdate}
               onCancel={() => setShowUpdate(false)}
             />
           )}
+          {showRedirect && (
+            <Alert
+              type="warning"
+              title="Redirect Patient?"
+              message="Patient data will be saved and you will be prompted to select a new destination."
+              cancel="Keep status"
+              destructive="Update status"
```

Comment:
These labels are incorrect per the Figma designs:

<img width="269" alt="Screen Shot 2021-07-14 at 7 08 10 PM" src="https://user-images.githubusercontent.com/140766/125716829-adcf9f54-fbd2-4a98-9814-6c822d42f090.png">